<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.3.2 (Python 3.13.7 on win32)" generated="2025-09-30T00:41:51.665791" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\okopo4ok\.cursor\extensions\robocorp.robotframework-lsp-1.13.0-universal\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-09-30T00:41:51.699497" elapsed="2.696578"/>
</kw>
<status status="PASS" start="2025-09-30T00:41:51.698974" elapsed="2.697370"/>
</test>
<status status="PASS" start="2025-09-30T00:41:51.667253" elapsed="2.729818"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="Robot Interactive Console" id="s1" pass="1" fail="0" skip="0">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-09-30T00:41:51.659908" level="WARN">Error in file 'c:\Users\okopo4ok\.cursor\extensions\robocorp.robotframework-lsp-1.13.0-universal\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
